vue/sbc-common-components/src/vuelidate.d.ts

```diff
@@ -1,35 +0,0 @@
-//
```

Comment:
Why not just delete this file?

Reply:
Weird, I did!

Reply:
Oh, the diff page says it was deleted. I think the diff of the lines all gone is just GutHub's way of showing you the contents of the deleted file.